protocol/messenger_communities.go

```diff
@@ -1432,11 +1494,53 @@ func (m *Messenger) DeclineRequestToJoinCommunity(request *requests.DeclineReque
 		return nil, err
 	}
 
-	err := m.communitiesManager.DeclineRequestToJoin(request)
+	dbRequest, err := m.communitiesManager.GetRequestToJoin(request.ID)
+	if err != nil {
+		return nil, err
+	}
+
+	community, err := m.communitiesManager.DeclineRequestToJoin(dbRequest)
 	if err != nil {
 		return nil, err
 	}
 
+	if community.IsControlNode() {
```

Comment:
If it's an admin, it should send the message to other privileged users too, so that they can either put it in pending declined or decline it if it's the control node.

Reply:
they will change the state during handling community events. It is already implemented and we have tests for this scenario